src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -35,9 +35,10 @@ const _ = require('lodash');
       });
 
       $http({ method: 'GET', url: self.linksMapping[partstat] }).then(function() {
-        var participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
+        const participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
```

Comment:
This line of code is not easy to understand. Better if there is a comment to expalain the structure of `partstat`.